contrib/gitian/gitian-build.py

```diff
@@ -5,6 +5,9 @@
 import subprocess
 import sys
 
+gsigs = 'https://github.com/monero-project/gitian.sigs.git'
+gbuild = 'https://github.com/devrandom/gitian-builder.git'
```

Comment:
`gbuild` is probably a bit confusing, since there is another script with the same that is being run later on.

Reply:
I renamed it to gbrepo.